README.md

```diff
@@ -62,6 +66,35 @@ You must run [Azure/login](https://github.com/Azure/login) before this action.
     cluster-name: "<cluster name>"
 ```
 
+### Non Admin User Example
+

```

Comment:
also explain that they need to install kubelogin and give an example

Reply:
done